openapi.yaml

```diff
@@ -0,0 +1,827 @@
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ApiResponse'
+        '500':
+          description: Internal server error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ApiResponse'
+  '/tasks/{taskId}/stop':
+    post:
+      tags:
+        - tasks
+      summary: Stop a task
+      operationId: stopTask
+      parameters:
+        - name: taskId
+          in: path
+          description: ID of the task to stop
+          required: true
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: Task successfully stopped
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  success:
+                    type: boolean
+                  message:
+                    type: string
+                  data:
+                    $ref: '#/components/schemas/Task'
+        '400':
+          description: Bad request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ApiResponse'
+        '500':
+          description: Internal server error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ApiResponse'
+components:
+  schemas:
+    Address:
+      type: object
+      properties:
+        type:
+          type: string
+        first_name:
+          type: string
+        last_name:
+          type: string
+        address_line_1:
+          type: string
+        address_line_2:
+          type: string
+        city:
+          type: string
+        state:
+          type: string
+        postal_code:
+          type: integer
+        country:
+          type: string
+        email_address:
+          type: string
+        phone_number:
+          type: string
+    Proxy:
+      type: object
+      properties:
+        ip_address:
+          type: string
+        port:
+          type: integer
+        protocol:
+          type: string
+        username:
+          type: string
+        password:
+          type: string
+    Task:
+      type: object
+      properties:
+        site_id:
+          type: integer
+        url:
+          type: string
+          format: uri
+        product_code:
+          type: string
+        style_index:
+          type: integer
+        size:
+          type: number
+          format: double
+        shipping_speed_index:
+          type: integer
+        billing_address_id:
+          type: integer
+        shipping_address_id:
+          type: integer
+        notification_email_address:
+          type: string
+          format: email
+    ApiResponse:
+      type: object
+      properties:
+        success:
+          type: boolean
+          format: int32
```

Comment:
Any reason for `int32` and not `int64` here? Just for consistency across the schema.

Reply:
Oops, that was a leftover from the previous/copied property. Removed the format there.